airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ConnectorDocumentationLayout.module.scss

```diff
@@ -6,6 +6,17 @@
   position: relative;
 }
 
+/*
+* This eliminates flickering scrollbars when resizing using the reflex splitter.
+* See more details in https://github.com/airbytehq/airbyte/pull/15996#issuecomment-1229019827
+*/
+.leftPanelStyle.leftPanelStyle {
```

Comment:
Repeated selectors were necessary here so that these took precedence over the `reflex-container > .reflex-element` styles